src/core/config/OperationConfig.js

```diff
@@ -4018,6 +4019,29 @@ const OperationConfig = {
         inputType: "string",
         outputType: "number",
         args: []
+    },
+    "Set Operations": {
```

Comment:
I think it would make more sense to have separate operations for each function (i.e. one op for 'Union', one for 'Intersection', one for 'Set Difference'...). This is what we've done for the arithmetic ops ('Sum', 'Mean' etc.).